### PR TITLE
WIP First option

### PR DESCRIPTION
**Pull request by inetic**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/510)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @benjaminbollen

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: examples/simple_key_value_store.rs [hunk 1]
```diff
@@ -80,22 +80,18 @@ use routing::utils::{encode, decode, public_key_to_client_name};
...
 
 Options:
-  -f, --first  Node runs as the first passive node in the network.
-  -n, --node   Node runs as a non-first, passive node in the network.
+  -n, --node   Run as a non-interactive routing node in the network.
   -h, --help   Display this help message.
 
-  Running without any args (or with only peer endpoints) will start an
-  interactive node.  Such a node can be used to send requests such as 'put' and
+  Running without the --node option will start an interactive node.
```
Comment (benjaminbollen) [inline examples/simple_key_value_store.rs]: ...  will start as a non-interactive node

File: src/routing_client.rs [hunk 2]
```diff
@@ -212,11 +213,14 @@ impl<F> RoutingClient<F> where F: Interface {
     /// or use CRUST self-discovery options.
     pub fn bootstrap(&mut self) -> Result<(), RoutingError> {
         // FIXME(ben 24/07/2015) this should become part of run() with integrated eventloop
-        let _ = self.connection_manager.start_accepting(vec![]);
+        println!("start accepting");
+        try!(self.connection_manager.start_accepting(vec![]));
```
Comment (benjaminbollen) [inline src/routing_client.rs]: good catch !

**Comment by benjaminbollen** [comment]
looks good to me

**Comment by benjaminbollen** [comment]
looks good to me, failing tests, but merging this and #509 to have a clear view on what needs patching

**Pull request merged**